packages/drawer/src/react/index.js

```diff
@@ -0,0 +1,155 @@
+import 'element-closest'
+import Collapsible from './collapsible';
+import core from '@pluralsight/ps-design-system-core'
+import * as glamor from 'glamor'
+import glamorous, { Div } from 'glamorous'
+import { transparentize } from 'polished'
+import Icon, { sizes as iconSizes } from '@pluralsight/ps-design-system-icon/react'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/react'
+import { bool, func, node, string } from 'prop-types'
+import React from 'react'
+
+const { gray01, gray02, gray03, gray04, gray06, white } = core.colors
+
+const DrawerBase = glamorous.div(
+  {
+    position: 'relative',
+    cursor: 'pointer',
+    transition: `background ${core.motion.speedNormal}`,
+    ':hover': {
+      background: transparentize(0.9, gray02)
+    }
+  },
+  ({ isOpen }) => isOpen
+    ? {
+      background: transparentize(0.8, gray02),
+      ':hover': {
+        background: transparentize(0.8, gray02)
+      }
+    }
+    : {}
+)
+
+const DrawerPanel = glamorous.div(
+  {
+    background: transparentize(0.94, gray02),
+    transition: `box-shadow ${core.motion.speedNormal}`
+  },
+  ({ isOpen, themeName }) =>(isOpen
+    ? { boxShadow: `inset 0 1px 3px 0 rgba(0,0,0,${themeName === themeNames.dark ? '0.5' : '0.1'})` }
+    : {}
+  )
+)
+
+const RowContainerWrapper = glamorous.div(
+  ({ themeName }) => ({
+    borderTop: `1px solid ${themeName === themeNames.dark ? gray04 : gray01 }`
+  })
+)
+const RowContainer = (props, context) => (
+  <RowContainerWrapper themeName={context.themeName}>
+    <Div marginTop={-1} padding={`0 57px 0 ${core.layout.spacingMedium}`}>
```

Comment:
What's the magic 57px for?

Reply:
It's the padding needed to push the right-most action button in the `Row` to the correct position in the mocks. We're actually tweaking this and it'll probably change. We'd love to be able to get this to a `core.layout.spacing` number.